src/UiTPASService/Controller/GetCardSystemsFromOrganizerRequestHandler.php

```diff
@@ -24,7 +24,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
     {
         $eventId = (new RouteParameters($request))->getOrganizerId();
```

Comment:
Maybe it's best to rename the variable to $organizerId

Reply:
Good remark!